src/builder.rs

```diff
@@ -130,4 +132,25 @@ pub(crate) mod test {
         let [i1] = dfg_builder.input_wires_arr();
         dfg_builder.finish_prelude_hugr_with_outputs([i1]).unwrap()
     }
+
+    pub(crate) fn closed_dfg_root_hugr(signature: FunctionType) -> Hugr {
+        let mut hugr = Hugr::new(NodeType::pure(ops::DFG {
```

Comment:
Exactly - I'll add a comment

Reply:
Not exactly to skip inference, but so that inference finds a solution for the whole graph and doesn't need to return extra answers to be passed into validation